src/applications/appeals/996/tests/hlr-contact-loop.cypress.spec.js

```diff
@@ -135,9 +135,9 @@ describe('HLR contact info loop', () => {
       `${BASE_URL}/edit-contact-information-mobile-phone`,
     );
 
-    cy.findByLabelText(/mobile phone/i)
-      .clear()
-      .type('8885551212');
```

Comment:
this is a cypress error now, saying its unsafe to do multiple chaining of commands